components/integrations/abstract_outgoing_webhook.tsx

```diff
@@ -142,94 +115,95 @@ export default class AbstractOutgoingWebhook extends React.PureComponent {
             return;
         }
 
-        const callbackUrls = [];
-        for (let callbackUrl of this.state.callbackUrls.split('\n')) {
-            callbackUrl = callbackUrl.trim();
+        const callbackUrls: string[] = [];
+        if (this.state.callbackUrls !== undefined) {
```

Comment:
suggestion : you can return early if `callbackUrls` is not defined, and then we dont have to indent changes here